src/Domain/Infra/DependencyInjection/Compiler/DoctrineObjectFieldMappingPass.php

```diff
@@ -30,6 +31,7 @@ public function __construct(string $tagName = 'msgphp.doctrine.object_field_mapp
     public function process(ContainerBuilder $container): void
     {
         $mappings = [];
+        $config = $container->get(MappingConfig::class);
```

Comment:
i know it works, but nevertheless we cant do it :) we should never "get" a service during compile. Here we're still in a pass thus executed during compiling.

I see we're struggling a bit with constructing the config, so i prosose to use `MappingConfig::fromArray()` everywhere. Here we'd use the parameter again (`fromArray($container->getParameter(...))`).